### DISCUSSION TITLE
O26 tutorial / examples?

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#156] [General]
Is there any documentation other than that in the systems programmer's instant and sources?

**Comment by wschaub** [comment]
system programmers instant is all I know for O26.  The analysis handbook might have something on using O26 as well. 

I basically use it like this:
```
X.DIS.
USER(username,password)
GET,FILENAME.
O26.
```

Once inside O26 I use
READ.FILENAME.

from there you can use ( and ) to move up and down a line and when you hit space you're inserting a line at the current 
position. 

 the . key will delete a line just under the current position

to save the file i make sure I'm backspaced all the way out and then type.
DIS.
once back in DIS I do  REPLACE,FILENAME.

this works particularly well for files that live under SYSTEMX that I want to quickly edit from the console. 


I use O26 in this video to add a line to the INPUT file while showing off MACE 6.0:
https://youtu.be/qm48yD6VEpU?si=JIAweSQO0JY-ORoN&t=194

**Comment by kej715** [comment]
Some additional O26 commands:

RO,  : read the OUTPUT file into O26

RI,filename. : same as READ,filename.

RFR. : read first record of current file

RNR. : read next record of current file

XDIS. : exit O26 w/o saving changes

HOLD. : suspend O26 and return to DSD. Use ASSIGN,jsn,1. to return to O26
from DSD.

DROP. : drop O26 and return to DSD.

0 : jump to beginning of buffer

1 : jump to 4th line

2 : jump to 8th line

3 : jump to 12the line

4 : jump to 16th line

5 : 20th

6 : 24th

7 : 28th

8 and 9 seem to mark the beginning and end of a block -- it's not clear
what can be done afterward ???
